doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -330,6 +330,8 @@ Libraries for networking
 * :ref:`lib_nrf_cloud_rest` library:
 
   * Updated to use the :ref:`lib_rest_client` library for REST API calls.
+  * Added :c:func:`nrf_cloud_rest_send_device_message` function that sends a JSON message to nRF Cloud using its SendDeviceMessage endpoint.
+  * Added :c:func:`nrf_cloud_rest_send_location` function that sends an NMEA sentence to nRF Cloud as a "GPS" type device message.
```

Comment:
```suggestion
  * Added :c:func:`nrf_cloud_rest_send_location` function that sends an NMEA sentence to nRF Cloud as a GPS-type device message.
```

Reply:
Cleared my 'requested changes'. I approve once these small changes are put in, and don't want to block merging since it'll likely be merged when I'm not here due to time zones.